### ISSUE TITLE
Feature request - Collapsible code samples

### ISSUE THREAD

**Issue by emajo** [rapi-doc/RapiDoc#865]
If the code samples are big they take a lot of the page space making the user navigation experience a lot worse, it would be nice to have the examples section collapsible (especially for the read render style), so the user can expand only the examples he actually needs.

**Comment by mrin9** [comment]
good suggestion, I will think about it, but expect any thing only after couple of months

**Comment by emajo** [comment]
Hi @mrin9, are there any updated about this?

Thanks,
Emanuele.
